src/modules/rpc/shared/handle-pending-transactions.ts

```diff
@@ -66,7 +80,7 @@ export async function tryIncrTransactionGasPrice(
     Logger.log('Try Replace Transaction', transaction.txHash);
 
     try {
-        const coefficient = 1.1;
+        const coefficient = 2;
```

Comment:
@rsmarques nao tenho a certeza em relação a isto, mas acho que pode ajudar quando for preciso aumentar o gas